example/vm/config.yaml

```diff
@@ -0,0 +1,201 @@
+    prometheus/self:
+        config:
+            scrape_configs:
+              - job_name: otel-collector
+                scrape_interval: 5s
+                static_configs:
+                - labels:
+                    collector_name: sn-collector
+                  targets:
+                    - 0.0.0.0:8888
+    hostmetrics:
+    # Hostmetrics scrapes metrics from various host systems.
+    # For more details, see
+    # https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/README.md
+        collection_interval: "30s"
+        root_path: /hostfs
+        scrapers:
+            cpu:
+                metrics:
+                    system.cpu.utilization:
+                        enabled: true
+            disk: {}
+            # cpu and disk are only supported if build with CGO_ENABLED=1
+            load: {}
+            filesystem:
+                metrics:
+                    system.filesystem.utilization:
+                        enabled: true
+                exclude_mount_points:
+                    match_type: regexp
+                    mount_points:
+                        - /dev/.*
+                        - /proc/.*
+                        - /sys/.*
+                        - /run/k3s/containerd/.*
+                        - /var/lib/docker/.*
+                        - /var/lib/kubelet/.*
+                        - /snap/.*
+                exclude_fs_types:
+                    match_type: strict
+                    fs_types:
+                        - autofs
+                        - binfmt_misc
+                        - bpf
+                        - cgroup2
+                        - configfs
+                        - debugfs
+                        - devpts
+                        - devtmpfs
+                        - fusectl
+                        - hugetlbfs
+                        - iso9660
+                        - mqueue
+                        - nsfs
+                        - overlay
+                        - proc
+                        - procfs
+                        - pstore
+                        - rpc_pipefs
+                        - securityfs
+                        - selinuxfs
+                        - squashfs
+                        - sysfs
+                        - tracefs
+            memory:
+                metrics:
+                    system.memory.utilization:
+                        enabled: true
+            # paging:
+            # processes:
+            # process:
+            network: {}
+    otlp:
+    # OTLP is the default protocol supported by all language implementations.
+    # For more details, see
+    # https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/otlpreceiver/README.md
+        protocols:
+            grpc:
+                endpoint:
+            http:
+                endpoint:
+processors:
+    # We recommend use of the batch processor.  We recommend the settings
```

Comment:
Let's also add a description of the batch processor:

> The batch processor receives telemetry data and batches it before sending it to the exporter. It improves the compression of the data and reduces the number of calls to emit the data from the Collector.

Also, should we include this link: https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor#batch-processor